index.html

```diff
@@ -0,0 +1,31 @@
+<html>
+
+<head>
+
+    <script src="script.js"></script>
+    <link rel="stylesheet" href="styles.css">
+
+    <!doctype html>
+    <html lang="en">
```

Comment:
These wouldn't be inside the `<head>` as the `doctype` declares the type of document and has to be the first line and the `html` has to encase the entire HTML structure which you already have a tag for.